core/templates/core/index.html

```diff
@@ -3,5 +3,16 @@
 {% block title %}Strona Główna{% endblock %}
 
 {% block content %}
-    <p class="text-3xl font-bold underline text-red-600">Gwardia Czapli</p>
+    <div class="flex items-center justify-center h-screen text-white">
+        <div class="text-center">
+            <h1 class="text-4xl text-white font-bold my-5">Gwardia Hub</h1>
+            <span>Strona internetowa, która ma służyć pomocą szkolną oraz efektywnym zarządzaniem aktywnościami Gwardii.</span><br/>
+            <span class="font-bold">W trakcie programowania!</span>
+            <br><br>
+            <div class="items-center">
+                <a href="https://example.com" class="hover:text-blue-700 text-gray-400 transition-colors duration-300"><i class="fa-brands fa-github fa-3x mr-5"></i></a>
```

Comment:
Change to correct link. Maybe modify hover color? Move `<i>` to other line that `<a>`

Reply:
I literally copied you code btw xd